src/coreclr/jit/compiler.h

```diff
@@ -9057,6 +9089,41 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return opts.compSupportsISA.HasInstructionSet(isa);
     }
 
+#ifdef DEBUG
+    //------------------------------------------------------------------------
+    // IsBaselineVector512IsaSupportedDebugOnly - Does the target have isa support required for Vector512.
+    //
+    // Returns:
+    //    `true` if AVX512F, AVX512BW and AVX512DQ are supported.
+    //
+    bool IsBaselineVector512IsaSupportedDebugOnly() const
+    {
+#ifdef TARGET_AMD64
+        return (compIsaSupportedDebugOnly(InstructionSet_AVX512F) &&
+                compIsaSupportedDebugOnly(InstructionSet_AVX512BW) &&
+                compIsaSupportedDebugOnly(InstructionSet_AVX512DQ));
+#else
+        return false;
+#endif
+    }
+#endif // DEBUG
+
+    //------------------------------------------------------------------------
+    // IsBaselineVector512IsaSupported - Does the target have isa support required for Vector512.
+    //
+    // Returns:
+    //    `true` if AVX512F, AVX512BW and AVX512DQ are supported.
+    //
+    bool IsBaselineVector512IsaSupported() const
+    {
+#ifdef TARGET_AMD64
+        return (compExactlyDependsOn(InstructionSet_AVX512F) && compExactlyDependsOn(InstructionSet_AVX512BW) &&
+                compExactlyDependsOn(InstructionSet_AVX512DQ));
```

Comment:
Same here. I think this can just be `compExactlyDependsOn(InstructionSet_Vector512)`

Reply:
Done